GeneratorInterface/RivetInterface/src/HiggsTemplateCrossSections.cc

```diff
@@ -85,6 +85,14 @@ namespace Rivet {
       return false;
     }
 
+    /// @brief Return true if particle decays to charged leptons.
+    bool ChLeptonDecay(const Particle &p) {
+      for (auto child : p.children())
+        if (PID::isChLepton(child.pid()))
```

Comment:
@adewit please set parentheses for bot the ```for``` loop and the ```if``` block to make the code readable in an unambiguous way

Reply:
Done - also made the same change to a few other functions which didn't have parentheses there either